src/Okta.Idx.Sdk/OktaVerify/OktaVerifyAuthenticationOptions.cs

```diff
@@ -0,0 +1,253 @@
+﻿// <copyright file="OktaVerifyAuthenticationOptions.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Okta.Idx.Sdk.Helpers;
+
+namespace Okta.Idx.Sdk.OktaVerify
+{
+    /// <summary>
+    /// A class that contains the options for authentication with Okta Verify.
+    /// </summary>
+    public class OktaVerifyAuthenticationOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OktaVerifyAuthenticationOptions"/> class.
+        /// </summary>
+        /// <param name="authenticationResponse">The authentication response.</param>
+        /// <param name="idxResponse">The Idx response.</param>
+        public OktaVerifyAuthenticationOptions(AuthenticationResponse authenticationResponse, IIdxResponse idxResponse)
+        {
+            this.AuthenticationResponse = authenticationResponse;
+            this.IdxContext = authenticationResponse.IdxContext;
+            this.StateHandle = idxResponse.StateHandle;
+            this.SelectAuthenticatorAuthenticateRemediationOption = idxResponse.FindRemediationOption(RemediationType.SelectAuthenticatorAuthenticate);
```

Comment:
We shouldn't expose raw remediation to the end user.

Reply:
I can mark the class sealed if we feel that's appropriate.